Library/Homebrew/download_strategy.rb

```diff
@@ -269,6 +269,17 @@ def initialize(name, resource)
 
   def fetch
     ohai "Downloading #{@url}"
+
+    url = resolve_url(@url)
```

Comment:
Might want to call this `resolved_url` or `actual_url` or something to make the below code a bit easier to follow